project1/css/index.css

```diff
@@ -62,3 +62,113 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+
+.container{
```

Comment:
Make sure you're using a linter! You want to keep your code as consistent as possible. A lot of people aren't happy that I'm being nitpicky with their spaces after the selectors, but it's really important that your code is always uniform! + linters will do it all for you automatically, so there's really no excuse to have mistakes like these. Definitely start linting your code and watch out for issues like these!